client/my-sites/email/email-providers-in-depth-comparison/data.tsx

```diff
@@ -11,6 +11,7 @@ import type { EmailProviderFeatures } from 'calypso/my-sites/email/email-provide
 const noop = () => {};
 
 export const professionalEmailFeatures: EmailProviderFeatures = {
+	slug: 'professional-email',
```

Comment:
Nit: we use `titan` and `google` for the slugs in the comparison page. Not sure if it's worth keeping this consistent.

Reply:
I think it doesn't hurt 😀. I will rename the ones from the Email Comparison page in another pull request.